pkg/kube/client.go

```diff
@@ -1325,10 +1324,7 @@ func FindIstiodMonitoringPort(pod *v1.Pod) int {
 	return 15014
 }
 
-// FilterIfEnhancedFilteringEnabled returns the namespace filter if EnhancedResourceScoping is enabled, otherwise a NOP filter.
+// FilterIfEnhancedFilteringEnabled returns the namespace filter .
 func FilterIfEnhancedFilteringEnabled(k Client) kubetypes.DynamicObjectFilter {
```

Comment:
nit: can remove this function entirely now and just call `k.ObjectFilter()` directly

Reply:
good call. https://github.com/istio/istio/pull/54093